pt-BR/mail/chrome/messenger/preferences/receipts.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY dialog.title                  "Confirmações de leitura">
```

Comment:
I received authorization to change every occurrence of "Return Receipt" to "confirmação de entrega".